crates/rpc/rpc/src/eth/api/traits/transaction.rs

```diff
@@ -1,18 +1,18 @@
 //! Database access for `eth_` transaction RPC methods. Loads transaction and receipt data w.r.t.
 //! network.
 
-use crate::eth::{
-    error::{EthApiError, EthResult},
-    revm_utils::EvmOverrides,
-    TransactionSource,
-};
+use std::sync::Arc;
+
```

Comment:
```suggestion
```

Reply:
removing the whitespace makes `fmt` order the `std` deps alphabetically with the other deps. I think it's very helpful to see which `std` deps are used easily by having it up top. kind of, in order of generality, since `std` lib is dep of most other deps.
```rust
use std::*;

use reth_dep::*;

use crate::*;

use super::*;
```